src/NUnitFramework/tests/Constraints/DictionaryContainsValueConstraintTests.cs

```diff
@@ -96,6 +96,14 @@ public void UsingPropertiesComparerIsHonored()
             Assert.That(dictionary, Does.ContainValue(value).UsingPropertiesComparer());
         }
 
+        [Test]
+        public void UsingCustomComparerIsHonored()
+        {
+            var dictionary = new Dictionary<string, int> { { "a", 1 }, { "b", 2 }, { "c", 3 } };
+
+            Assert.That(dictionary, new DictionaryContainsValueConstraint("1").Using<int, string>((left, right) => left.ToString() == right));
```

Comment:
Use standard naming for the parameters:

```suggestion
            Assert.That(dictionary, new DictionaryContainsValueConstraint("1").Using<int, string>((actual, expected) => acual.ToString() == expected));
```